requirements-dev.txt

```diff
@@ -18,4 +18,4 @@ cmake
 custatevec-cu12
 pylint
 scipy~=1.12.0
-quimb
+git+https://github.com/jcmgray/quimb.git
```

Comment:
Do we have a timeline for when the changes we want will be pushed to a stable quimb release?

Reply:
I am not sure since `quimb` doesn't seem to be released on a regular schedule, but I'll change this line as soon as `quimb 1.8.1` is released (there's a story about this in the second epic)